objects/ohitbox/Step_2.gml

```diff
@@ -466,16 +468,18 @@ else
 						var isP2 = (owner.playerOwner.playerID == 2);
 					
 						ProcessHit(counterHitProperty, collision_list[| i]);
-					
+						
 						// Display counterhit text
 						if (other.counterHitProperty.CounterHitLevel == 1)
 						{
+							audio_play_sound(sfx_SmallCounter, 0, false);

```

Comment:
Why is this outside of `with (counterParticle)` unlike all of the other counterhit sounds? Make them all match for consistency unless there is some reason why this is set up this way.

Reply:
Good question, i don;t recall if there was a reason i did that but i'll do that right away